qiskit/extensions/standard/cy.py

```diff
@@ -17,20 +17,20 @@
 """
 controlled-Y gate.
 """
-from qiskit.circuit import Gate
+from qiskit.circuit import ControlledGate
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit import QuantumRegister
 from qiskit.extensions.standard.s import SGate
 from qiskit.extensions.standard.s import SdgGate
 from qiskit.extensions.standard.cx import CnotGate
 
 
-class CyGate(Gate):
+class CyGate(ControlledGate):
     """controlled-Y gate."""
 
     def __init__(self):
         """Create new CY gate."""
-        super().__init__("cy", 2, [])
+        super().__init__("cy", 2, [], num_ctrl_qubits=1)
 
```

Comment:
add `self.base_gate` here for the CyGate?

Reply:
Done